packages/venia-ui/lib/components/SearchBar/__stories__/searchBar.js

```diff
@@ -3,8 +3,13 @@ import { storiesOf } from '@storybook/react';
 import SearchBar from '../searchBar';
 import defaultClasses from '../searchBar.css';
 
-const stories = storiesOf('SearchBar', module);
+const stories = storiesOf('Components/SearchBar', module);
 
 stories.add('Search Bar', () => (
-    <SearchBar classes={defaultClasses} isOpen={true} />
+    <SearchBar
+        classes={defaultClasses}
+        isOpen={true}
+        location={{}}
+        history={{ push: () => {} }}
```

Comment:
Would discourage from doing this. This will create a new function on every re-render.